src/geometry/geom2/canonicalize.test.js

```diff
@@ -0,0 +1,31 @@
+const test = require('ava')
+
+const {canonicalize, fromPoints, toString} = require('./index')
+
+test('canonicalize: Updates a populated geom2 with canonalized sides', (t) => {
+  const points = [[0, 0], [1, 0], [0, 1]]
+  const expected = {
+    baseSides: [
```

Comment:
what are basesides ?

Reply:
basePoints, baseSides, basePolygons have been removed.
i tested with the new primitives as well. :)